.github/workflows/test.yml

```diff
@@ -33,6 +33,43 @@ jobs:
             inst/validation
           path: templates
 
+      - name: Overwrite quality control scripts with dv.templates ones 🧹
+        shell: bash
+        run: |
+          TEMPLATE_DIR="templates/inst/validation"
+          REPO_DIR="inst/validation"
+
+          if test ! -d "${REPO_DIR}"; then
+            echo "### Quality Control documentation is not present"
+            echo "### Include quality control documentation or skip it by creating following file 'inst/validation/skip_qc'"
+            exit 1
+          fi
+
+          if test -f "${REPO_DIR}/skip_qc"; then
+            echo "skip_qc file found skipping qc documentation"
+            exit 0
+          fi
+          
+          # Conserve specs.R file
+          cp "${REPO_DIR}/specs.R" "${TEMPLATE_DIR}/specs.R"
+
+          set +e  # Disable immediate exit in conflicting parts on error for diff command
+          DIFF_OUTPUT=$(diff -qr "${TEMPLATE_DIR}" "${REPO_DIR}")
+          set -e  # Enable immediate exit on error after diff command
+
+          echo "# Quality control files" | tee -a "${GITHUB_STEP_SUMMARY}"
+          
+          if [ -n "${DIFF_OUTPUT}" ]; then
+            echo "Warning: The contents of the directories are different!" | tee -a "${GITHUB_STEP_SUMMARY}"
+            echo "Modified files:" | tee -a "${GITHUB_STEP_SUMMARY}"
+            echo "$DIFF_OUTPUT" | tee -a "${GITHUB_STEP_SUMMARY}"
+          else
+            echo "The directories are identical." | tee -a "${GITHUB_STEP_SUMMARY}"
```

Comment:
like the check for code drift. but if the REPO_DIR will be removed anyways, why require/expect to have it in the target repo in the first place?

My approach would be something like: if there is a specs.R file in the path, then copy the TEMPLATE_DIR code from dv.templates and run it there. If not assume skip_qc and move on.

Reply:
In this case I would rather have an explicit skip in the qc validation. This way we remind developers that qc is a requisite and makes skipping an explicit action. The only thing we force in the end is that there is a validation folder either with `specs.R` or with `skip_qc`.

I do this with a file so it is something that we can change with a simple push in the repo (I assume with a variable you mean an env variable defined at the repo level). Also, it seems more obvious than a variable in the repo that is less transparent.